developers/limits.md

```diff
@@ -8,4 +8,33 @@ description: >-
 
 Tableland isn't a new database — it’s just Web3-native relational tables. This means there are no limits to the new types of interactions and experiences you can build on Tableland. But with great power comes great responsibility! For now, each table is limited to the following constraints:
 
-<table><thead><tr><th>Constraint</th><th data-type="number">Max Size</th></tr></thead><tbody><tr><td>Rows</td><td>100000</td></tr><tr><td>Columns</td><td>24</td></tr><tr><td>Text Length</td><td>1024</td></tr></tbody></table>
```

Comment:
Is there a reason the HTML table was not formatted?

Reply:
> Is there a reason the HTML table was not formatted?

Maybe this is a result of some kind of markdown client? Regardless this seems like an improvement